DentOS_Framework/DentOsTestbed/src/dent_os_testbed/utils/test_utils/tgen_utils.py

```diff
@@ -407,6 +407,7 @@ async def tgen_utils_setup_streams(device, config_file_name, streams, force_upda
                 input_data=[{device.host_name: [{"name": s, "pkt_data": streams[s]}]}]
             )
             device.applog.info(out)
+            assert out[0][device.host_name]["rc"] == 0, f"Setting tgen traffic failed.\n{out}"
```

Comment:
don't forget that changes in this file (and other files) belong to a different PR and should not be included in the bridging tests PR